app/views/students/attendances/_attendance_records_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= form_with(method: :get, url: url) do |form| %>
```

Comment:
I'd say this and [_attendance_stats.html.erb](https://github.com/bryanmg/ammdk_colima/pull/43/files#diff-1589a313ff1c39edd9df4e6bc936335a1ecbeb8ac1c4ca79be2dbc45df99da26) changes had to belong to another PR 👀

Reply:
You're could be, but it is time to deploy :(